spec/client_spec.rb

```diff
@@ -1,44 +1,36 @@
-require 'spec_helper'
+require "spec_helper"
 
 include Twingly::Search
 
 describe Client do
-  subject { Client.new('api_key') }
+  let(:valid_api_key) { "api_key" }
 
   describe ".new" do
-    context 'with API key as arguments' do
-      it { should be_a Client }
-    end
+    context "with API key as argument" do
+      subject { described_class.new(valid_api_key) }
 
-    it "BASE_URL should be parsable" do
-      expect(URI(Client::BASE_URL).to_s).to eq(Client::BASE_URL)
+      it { should be_a Twingly::Search::Client }
```

Comment:
I just thought it looks weird checking that `described_class.new` is a `described_class`. It's easier to understand if its explicit, but the "correct way" would probably be to use `described_class` here too.